src/lambda_codebase/initial_commit/bootstrap_repository/adf-bootstrap/example-scp.json

```diff
@@ -12,13 +13,8 @@
         },
         {
             "Effect": "Deny",
-            "Action": [
-                "config:DeleteConfigRule",
-                "config:DeleteConfigurationRecorder",
-                "config:DeleteDeliveryChannel",
-                "config:Stop*"
-            ],
+            "Action": ["config:DeleteConfigRule", "config:DeleteConfigurationRecorder", "config:DeleteDeliveryChannel", "config:Stop*"],
```

Comment:
Personally I don't like this change, as it makes it harder to add a new line and check the different actions that are supported. Can we change the linter config to prefer lists on new lines per items?

Reply:
I agree, I'll create a config for this and revert.

Reply:
Changed this for all files in latest commit. Resolving.